protocol/block_test.go

```diff
@@ -97,7 +97,7 @@ func TestWaitForBlockSoonWaits(t *testing.T) {
 	makeEmptyBlock(t, c) // height=2
 
 	go func() {
-		time.Sleep(10 * time.Millisecond) // sorry for the slow test 😔
+		time.Sleep(10 * time.Millisecond) // sorry for the slow test 
```

Comment:
Fixed in https://github.com/chain/chain/pull/261/commits/8283533fcc9a0404c2bf4f8101a1777b95e50028